### ISSUE TITLE
FPS meter

### ISSUE THREAD

**Issue by techtonik** [gdamore/proxima5#27]
Would be nice to have one, maybe in window status.

**Comment by gdamore** [comment]
Yeah; the challenge is measuring that; to do so we need to export a drain API from tcell.  I started this, but didn't finish.  That said, proxima5 doesn't ever run at maximum speed; there are some small sleeps injected periodically.

**Comment by techtonik** [comment]
One of the first programming principles I learned more than 10 years ago was "make it optional". It looks like it is still actual. =)
